test/terraform/provider.go

```diff
@@ -18,3 +19,19 @@ func InitTestAwsProvider(providerLibrary *terraform.ProviderLibrary, version str
 	providerLibrary.AddProvider(terraform.AWS, provider)
 	return provider, nil
 }
+
+func InitTestGithubProvider(providerLibrary *terraform.ProviderLibrary, version string) (*github.GithubTerraformProvider, error) {
```

Comment:
Shouldn't this be inside `fake_github_provider.go` like terraform ?

Reply:
I copied the way we do it for aws, as you can see the same fuction for aws is in the same file

Reply:
Yeah we have cycle dependencies that's why this code remain here, but I agree it should be in another place since it test purpose logic. Maybe we can discuss this when the split will be done.

Reply:
Yeah this is in a test package so this is fine